tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1053,10 +1062,11 @@ def test_positive_end_to_end(self):
                 })
                 break
         # step 2.15.1: Enable product content
-        activation_key.content_override(data={'content_override': {
-            u'content_label': AK_CONTENT_LABEL,
-            u'value': u'1',
-        }})
+        if setting_is_set('fake_manifest'):
```

Comment:
Calling at the module level is not good because it will introduce import time side effects like reading the config file.

Maybe adding it to the setupclass or something like that would be better. I will update the PR